docs/src/prototypes/chatMessages/ChatMessageWithPopover/ReactionPopup.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react'
+import keyboardKey from 'keyboard-key'
+import { Popup, Menu, Reaction, ReactionProps } from '@stardust-ui/react'
+
+const getAriaLabel = ({ content: numberOfPersons, icon: emojiType }: ReactionProps) => {
+  if (numberOfPersons === 1) {
+    return `One person reacted to this message with a ${emojiType} emoji. Open menu to see person who reacted.`
+  }
+  return `${numberOfPersons} people reacted this message with a ${emojiType} emoji. Open menu to see people who reacted.`
+}
+
+class ReactionPopup extends React.Component<ReactionProps, { open: boolean }> {
+  state = {
+    open: false,
+  }
+
+  handleKeyDownOnMenu = e => {
+    if (e.keyCode === keyboardKey.Tab) {
```

Comment:
why not close the popup menu on blur?

Reply:
blur is invoked even if I press arrow keys (up/down), so I was not able to use this event.